docs/public_nodes.md

```diff
@@ -0,0 +1,36 @@
+The public query tool at [query.neurobagel.org](https://query.neurobagel.org)
+queries the public Neurobagel federation API at [federate.neurobagel.org](https://federate.neurobagel.org)
+which provides access to all publicly accessible Neurobagel nodes.
+
+To start running your own cohort queries, 
+all you have to do is visit [query.neurobagel.org](https://query.neurobagel.org),
+enter your cohort criteria into the web interface, and click the "Submit" button.
+
+## Public Neurobagel Nodes
+
+At the moment, the following public Neurobagel nodes are available:
```

Comment:
🍒 For a more concise way to indicate how to select the node, how about adding this:

```suggestion
At the moment, the following public Neurobagel nodes are available 
(you can query a specific node by selecting it from the dropdown under "Neurobagel graph" in the query tool):
```

and then removing the sentence "You can query this node by selecting..." from each node's bullet point below? 
Since the full names rather than abbreviations are used as node names anyways!